app/views/partials/_header.html.erb

```diff
@@ -1,14 +1,19 @@
-<div class="header">
-  <%= link_to  "Home", root_path, class: "btn btn-outline-dark" %>
-  <%= link_to  "Create recipe", new_recipe_path, class: "btn btn-outline-dark" %>
-
+<div class="header d-flex">
+  <div class="col">
+    <%= link_to  "Home", root_path, class: "btn btn-outline-dark" %>
+    <%= link_to  "Create recipe", new_recipe_path, class: "btn btn-outline-dark" %>
+    <% if current_user %>
+      <%= link_to  "My menu", menu_path(current_user.menu) , class: "btn btn-outline-dark" %> 
+    <% end %>
+  </div>
   
   <% if current_user %>
-    <%= current_user.email %>
-    <%= link_to  "My menu", menu_path(current_user.menu) , class: "btn btn-outline-dark" %> 
-    <%= link_to 'Log out', log_out_path, method: :delete, class: "btn btn-outline-dark"%>
-    <% else %>
-    <%= link_to  "Sign up", sign_up_path, class: "btn btn-outline-dark" %>
-    <%= link_to  "Log in", log_in_path, class: "btn btn-outline-dark" %>
-  <% end %>
+  <div class= "col d-flex align-items-center justify-content-end">
```

Comment:
This div,
```
<div class= "col d-flex align-items-center justify-content-end">
```
 is only going to be printed if current_user exists, maybe you would like to move that out from the if-statement?

Reply:
Also make sure the `else` is on the same indentation level as the `if`